### PR TITLE
[WIP] rh-che #395: Adding multi-tenant che template to dsaas-services

### PR DESCRIPTION
**Pull request by ibuziuk**
Signed-off-by: Ilya Buziuk <ibuziuk@redhat.com>

### CODE REVIEW

**Comment by ibuziuk** [comment]
blocked by https://github.com/redhat-developer/rh-che/pull/407

**Comment by ibuziuk** [comment]
@pbergene can not add you as a reviewer for some reason :/

File: dsaas-services/rh-che.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+services:
+- hash: 808bc0e456437ab757cda1f23e43c06cac694b4b
+  hash_length: 7
+  name: che
+  parameters:
+    IMAGE: push.registry.devshift.net/che/che-multiuser
```
Comment (vpavlin) [inline dsaas-services/rh-che.yaml]: Remove `push.` from registry URL and it should work

Comment (ibuziuk) [reply dsaas-services/rh-che.yaml]: thanks, lemme try

**Pull request merged**